samples/ship-loading/README.md

```diff
@@ -0,0 +1,62 @@
+---
+page_type: sample
+languages:
+- python
+author: frtibble
+ms.author: frances.tibble@microsoft.com
+products:
+- azure-qio
+- azure-quantum
+description: "Learn how to solve a binary optimization problem using the Azure Quantum optimization service"
+urlFragment: azure-quantum.job-shop-problem
+---
+
+# Shipping Sample
```

Comment:
For better SEO performance of rendered sample pages on MS Docs, add some brand or tech to disambiguate the H1. Like: Ship Loading Sample on Azure Quantum

Reply:
Will correct here as well, as per above discussion.